lib/src/state_machine.dart

```diff
@@ -134,7 +131,7 @@ class State extends Disposable implements Function {
 
   /// Determine whether or not this [State] is active.
   bool call([_]) {
-    return _machine.current == this;
+    return _machine!.current == this;
```

Comment:
I'm not sure if the wildcard state will ever be called here but that would be the only unsafe use of `!` here.